java/system-test/system-test-suite/src/main/java/sleeper/systemtest/suite/fixtures/SystemTestInstance.java

```diff
@@ -119,6 +120,7 @@ private static InstanceProperties createInstancePropertiesWithDefaults() {
         InstanceProperties properties = new InstanceProperties();
         properties.set(LOGGING_LEVEL, "debug");
         properties.set(RETAIN_INFRA_AFTER_DESTROY, "false");
+        properties.set(RETAIN_LOGS_AFTER_DESTROY, "false");
```

Comment:
To clarify, this also needs changing to address my previous comment: https://github.com/gchq/sleeper/pull/5965#discussion_r2494826704

> We need to keep the logs after a system test instance is destroyed, otherwise we wouldn't be able to see what happened in the nightly tests.